Formula/thrax.rb

```diff
@@ -14,10 +21,17 @@ class Thrax < Formula
 
   depends_on "openfst"
 
+  on_linux do
+    depends_on "gcc"
+    depends_on "python@3.9"
```

Comment:
Do we not need Python on macOS?

Reply:
I guess we can switch to Homebrew Python if preferred. The formula has been using macOS system python when it was recently added (v1.3.5).